### PR TITLE
feat: add first success-screen banner

### PR DESCRIPTION
**Pull request by Shou**
We add the first success-screen banner for Etcher, including buttons to
star and tweet the project.

Changelog-Entry: Add the first success-screen banner, with star and
tweet buttons.

### CODE REVIEW

**Comment by Shou** [comment]
Let me know what you think. And here's a screenshot.

image

**Review by konmouz — COMMENTED** [review]
I think both buttons should have the same width, now it looks a bit weird. Also, twitter logo and text looks a bit off to the right. @taahirisaacs any ideas?

File: pages/success-banner.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    made with
+    <img className="icon" src="/static/love.svg" />
+    by
+    <img className="brand" src="/static/resin.png" />
+  </footer>
+)
+
+const Page = ({ content }) => (
+  <html>
+    <Head>
```
Comment (lurch) [inline pages/success-banner.js]: Just out of curiosity - how come some tags are alllowercase and others are MixedCase?

**Comment by lurch** [comment]
Are some of the 'static' files the same between this repo and the main Etcher repo? If so, perhaps it'd make sense for the 'build' step to pull those files from the Etcher repo, rather than storing duplicate copies in this repo?

Also, the `twitter.png` image seems much larger (i.e. higher-resolution) than the `octocat.png` image?

File: pages/success-banner.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      </a>
+    )
+  }
+}
+
+const Banner = () => (
+  <main className="vertical center">
+    <div>
+      <h1>We hope you enjoyed using</h1>
+      <img src="/static/etcher.svg" className="etcher-logo"></img>
```
Comment (lurch) [inline pages/success-banner.js]: Should the `` tag be _inside_ the `<h1>` tag? https://stackoverflow.com/questions/1605454/can-we-place-img-inside-h1-according-to-web-standards

File: pages/success-banner.js (hunk 1 above)
Comment (craigmulligan) [reply pages/success-banner.js]: Lowercase = html tags and Uppercase = react components.

**Comment by Shou** [comment]
@lurch Sure! We could do that. Do you have any suggestions for how? One way I see is using `curl` and Github's "raw" links, hopefully it dishes out proper dates so it only downloads when they're updated.

File: pages/success-banner.js (hunk 2 above)
Comment (craigmulligan) [reply pages/success-banner.js]: I think you likely want to use the margin bottom from h1 as spacing so it's simpler (less css) to have it outside `h1`. Though no need to close the image tag.

File: package.json [hunk 3]
```diff
@@ -20,8 +20,9 @@
   "scripts": {
     "build:data": "babel-node lib/fetchData.js",
     "start": "next && node-sass -w --include-path scss styles/index.scss static/index.css",
-    "build": "npm run build:data && npm run build-css && next build",
+    "build": "npm run build:data && npm run build-css && npm run build-banner && next build",
```
Comment (craigmulligan) [inline package.json]: Could we add this to the css build task instead?

**Review by craigmulligan — APPROVED** [review]
lgtm.

**Comment by Shou** [comment]
Here we go.

@konmouz what do you think now?

image

**Comment by konmouz** [comment]
@Shou, looks much better! Can you also increase the space between logo and text on both buttons (github & twitter) by something like x1.5?

**Comment by Shou** [comment]
Sure! Like this?

image

**Comment by konmouz** [comment]
@Shou, thanks! Unless @taahirisaacs wants to make any changes, I think it is ready.

**Comment by Shou** [comment]
Cool! Now that we've churned away the small aberrations, @alexandrosm what do you think?

**Comment by alexandrosm** [comment]
@Shou almost perfect! It feels like the "we hope you enjoyed using etcher" part should have smaller letters. it's a bit shouty at the moment. Other than that, perfect, push it out and we can fix anything else on the fly. it's web content afterall!

**Comment by Shou** [comment]
Rebased to master to pass the tests.

@alexandrosm does this look better? Anyway, that's true, so I'll get it merged now.
image

**Comment by lurch** [comment]
Sorry for the late follow-up, but I've just noticed that this doesn't contain the Etcher version-number in the 'footer'. Wasn't that the whole point of adding the `etcher-version` query-param?

**Comment by Shou** [comment]
@lurch good catch, I've fixed that now in a new PR.

**Pull request merged**